jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile

```diff
@@ -84,16 +84,20 @@ pipeline {
     stage('Run tests') {
```

Comment:
Could you also get rid of this old thing, please ? It's just cluttering up the job output.

Reply:
Don't we want to keep it for running the `post-deploy-checks` test? (when adding it back)

Reply:
I don't think we'll use the same approach. We'll probably run those in sequence as well.

Reply:
done